README.md

```diff
@@ -171,10 +184,16 @@ String searching using the `_q` parameter is fuzzy by default (`*` on both ends
 As well, search terms that include spaces will `AND` each term instead of the implicit `OR`
 
 e.g. `full text` search string will be converted to the following query string `(*full AND text*)`
+
+Querying for nil fields will return documents that have `nil` or an empty array`[]` for the given key
+
 ```ruby
 # Query by full text string searching
 DummyElasticSearchModel.where(_q: "Hello")
 
+# Query by nil fields
+DummyElasticSearchModel.where(_q: { my_string: nil }) 
```

Comment:
Important note: the `nil` searching will be applied when passed like so
```
DummyElasticSearchModel.where(my_string: nil)
```

The above behaves like a filter, where as using the `_q` option behaves like fuzzy search.